modules/apps/translation/translation-service/src/main/java/com/liferay/translation/internal/importer/XLIFFInfoFormTranslationImporter.java

```diff
@@ -83,604 +32,25 @@
  */
 @Component(
 	property = "content.type=application/xliff+xml",
-	service = {
-		TranslationInfoItemFieldValuesImporter.class,
-		TranslationSnapshotProvider.class
-	}
+	service = TranslationInfoItemFieldValuesImporter.class
 )
 public class XLIFFInfoFormTranslationImporter
-	implements TranslationInfoItemFieldValuesImporter,
-			   TranslationSnapshotProvider {
-
-	@Override
-	public TranslationSnapshot getTranslationSnapshot(
-			long groupId, InfoItemReference infoItemReference,
-			InputStream inputStream)
-		throws IOException, PortalException {
-
-		return _getTranslationSnapshot(
-			groupId, infoItemReference, inputStream, true);
-	}
+	implements TranslationInfoItemFieldValuesImporter {
```

Comment:
@Valmir-unicap Please rename this class to match the name of the interface, we usually follow the pattern that for interface `A`, we name an implementation `*A`.

Here, `XLIFFTranslationInfoItemFieldValuesImporter`.